java/arcs/android/storage/database/DatabaseImpl.kt

```diff
@@ -385,18 +383,43 @@ class DatabaseImpl(
         return typeMap
     }
 
+    private fun Long.toBoolean() = when (this) {
+        0L -> false
+        1L -> true
+        else -> throw IllegalArgumentException(
+            "Could not convert $this to Boolean, expected 0 or 1."
+        )
+    }
+
+    private fun Boolean.toLong() = when (this) {
+        false -> 0L
+        true -> 1L
+    }
+
+    private fun Cursor.getBoolean(i: Int) = getLong(i).toBoolean()
+
+    private fun SQLiteProgram.bindBoolean(i: Int, value: Boolean) = bindLong(i, value.toLong())
```

Comment:
This are pretty great, maybe they could live in the `arcs.common.SqlUtils` file?   (would keep `Long.toBoolean`/`Boolean.toLong` private, and only have `Cursor.getBoolean` and `SQLiteProgram.bindBoolean` be public there)

Reply:
Done